src/locales/ko/mystery-encounters/fiery-fallout-dialogue.json

```
@@ -1,26 +1,26 @@
 {
-  "intro": "You encounter a blistering storm of smoke and ash!",
-  "title": "Fiery Fallout",
-  "description": "The whirling ash and embers have cut visibility to nearly zero. It seems like there might be some... source that is causing these conditions. But what could be behind a phenomenon of this magnitude?",
-  "query": "What will you do?",
+  "intro": "매캐한 연기와 화산재의 폭풍을 맞닥뜨렸다!",
+  "title": "불타는 폴아웃",
+  "description": "소용돌이치는 화산재와 불씨로 인해 앞이 거의 보이질 않습니다.\n이런 데는 무언가… 원인이 있는 것 같습니다.\n그렇지만 이정도로 큰 규모라면 뒤에 도대체 뭐가 있는 걸까요?",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Find the Source",
-      "tooltip": "(?) Discover the source\n(-) Hard Battle",
-      "selected": "You push through the storm, and find two {{volcaronaName}}s in the middle of a mating dance!$They don't take kindly to the interruption and attack!"
+      "label": "원인을 찾는다",
+      "tooltip": "(?) 원인을 밝힘\n(-) 어려운 배틀",
+      "selected": "화산재 폭풍을 뚫고 나아가자,\n구애의 춤을 추고 있는 {{volcaronaName}} 두 마리를 발견했다!$방해받고 싶지 않았던 두 포켓몬이 싸움을 걸어왔다!"
     },
     "2": {
-      "label": "Hunker Down",
-      "tooltip": "(-) Suffer the effects of the weather",
-      "selected": "The weather effects cause significant\nharm as you struggle to find shelter!$Your party takes 20% Max HP damage!",
-      "target_burned": "Your {{burnedPokemon}} also became burned!"
+      "label": "낮은 자세로 이동한다",
+      "tooltip": "(-) 날씨의 피해를 받음",
+      "selected": "쉘터를 향해 고분고투하는 동안\n재와 불씨로 인해 심각한 피해를 입었다!$모든 포켓몬이 HP가 20% 피해를 받았다!",
```

```suggestion
      "selected": "피할 만한 곳을 찾으며 고군분투하는 동안\n재와 불씨로 인해 심각한 피해를 입었다!$모든 포켓몬이 HP의 20% 피해를 받았다!",
```
표준 표기법에 맞게 "셸터" 혹은 "피할 곳", "피할 만한 곳" 정도가 좋지 않을까 제안합니다.

---

```suggestion
      "selected": "쉘터를 향해 고분고투하는 동안\n재와 불씨로 인해 심각한 피해를 입었다!$모든 포켓몬이 HP의 20% 피해를 받았다!",
```